### ISSUE TITLE
Valid ARIA 1.1 combobox incorrectly flagged for missing aria-controls attribute

### ISSUE THREAD

**Issue by ChipElavon** [dequelabs/axe-core#3141]
Product: axe-core

Expectation: axe-core recognizes ARIA 1.1-compliant combobox as valid

Actual: axe-core flags ARIA 1.1-compliant combobox as missing a required attribute

Motivation: To remove what seems to be a false positive from the axe-core results

<pre><code>
axe-core version: 4.3.1
axe DevTools Extension v4.12.2
Browser and Assistive Technology versions
macOS Chrome Version 92.0.4515.159 (Official Build) (x86_64)
</code></pre>

Here's a minimal example that will show the issue:
```
<html>
  <body>
    <div role="combobox" aria-expanded="false"><input type="text" /></div>
    <div role="combobox" aria-expanded="false" aria-owns="explicit-textbox"><input id="explicit-textbox" type="text" /></div>
  </body>
</html>
```
Open the above html in Chrome and scan the page with the axe DevTools Extension. You'll see the top combobox has a "Required ARIA attributes must be provided" violation for a missing `aria-controls` attribute. The bottom combobox does not get flagged for this issue.

Based on my reading of the spec the `aria-owns` attribute should not be required to make the combobox compliant. From the spec: [the combobox MUST "...contains or owns a text input element with role textbox..."](https://www.w3.org/TR/wai-aria-1.1/#combobox) and the definition of "owns" reads ["An 'owned element' is any DOM descendant of the element, any element specified as a child via aria-owns, or any DOM descendant of the owned child."](https://www.w3.org/TR/wai-aria-1.1/#dfn-owned-element) Since the first combobox in the example has a `textbox` descendant, the `aria-owns` attribute should not be required to show ownership of that element.

I believe this change in behavior was introduced when ARIA 1.2 combobox support was added in #3033.

This will negatively impact any users of the [material-ui autocomplete component](https://material-ui.com/components/autocomplete/#autocomplete) as it is being flagged as being out of compliance by axe scans.

**Comment by straker** [comment]
Thanks for the issue. With the change to the 1.2 combobox we dropped the requirements for owning elements, but added that `aria-controls` or `aria-owns` be present. Using the examples provided, both are missing the requirement that they own or control a `listbox` element. 

However, testing the [combobox element](https://material-ui.com/components/autocomplete/#combo-box) on material-ui, I'm not seeing any violations using axe-core@4.3.3 when either the combobox is collapsed (`aria-expanded=false`) or expanded.

**Comment by ChipElavon** [comment]
If axe-core@4.3.3 no longer reports an issue with the material-ui combobox then that should resolve this issue. Looks like the latest Chrome extension still uses axe-core@4.3.1 so I'll wait for a new release of the extension and try again.
Thanks!
